JavaScript/hw6_object/JS_object.js

```diff
@@ -0,0 +1,105 @@
+const positions = [
+  'Телепорт бытовой VZHIH-101',
+  'Отвертка ультразвуковая WHO-D',
+  'Ховерборд Mattel 2016',
+  'Нейтрализатор FLASH black edition',
+  'Меч световой FORCE (синий луч)'
+];
+
+const prices = [
+  10000,
+  4800,
+  9200,
+  2500,
+  57000
+];
+
+let hitName = positions[2];
+let hitPrice = prices[2];
+
+// Задача 1
+let hit = {};
+hit.name = hitName;
+hit.price = hitPrice;
+
+console.log(`Хит продаж мартобря: <${hit.name}> цена ${hit.price} Q`);
+
+// Задача 2
+let items = [];
+let pos1 = {};
+pos1.name = positions[0];
+pos1.price = prices[0];
+
+let pos2= {};
+pos2.name = positions[1];
+pos2.price = prices[1];
+
+let pos3 = {};
+pos3.name = positions[2];
+pos3.price = prices[2];
+
+let pos4 = {};
+pos4.name = positions[3];
+pos4.price = prices[3];
+
+let pos5 = {};
+pos5.name = positions[4];
+pos5.price = prices[4];
+
+items.push(pos1);
```

Comment:
please rewrite this like loop (for or forEach)
items.push({name: positions[i], price: prices[i]});